src/m365/purview/commands/threatassessment/threatassessment-list.spec.ts

```diff
@@ -0,0 +1,317 @@
+    "createdBy": {
+      "user": {
+        "id": "c52ce8db-3e4b-4181-93c4-7d6b6bffaf60",
+        "displayName": "Ronald Admin"
+      }
+    }
+  }];
+
+  const threatAssesmentEmailFileResponse: any = [{
+    "@odata.type": "#microsoft.graph.emailFileAssessmentRequest",
+    "id": "ab2ad9b3-2213-4091-ae0c-08d76ddbcacf",
+    "createdDateTime": "2019-11-20T17:05:06.4088076Z",
+    "contentType": "mail",
+    "expectedAssessment": "block",
+    "category": "malware",
+    "status": "completed",
+    "requestSource": "administrator",
+    "recipientEmail": "tifc@a830edad9050849EQTPWBJZXODQ.onmicrosoft.com",
+    "destinationRoutingReason": "notJunk",
+    "contentData": "",
+    "createdBy": {
+      "user": {
+        "id": "c52ce8db-3e4b-4181-93c4-7d6b6bffaf60",
+        "displayName": "Ronald Admin"
+      }
+    }
+  }];
+
+  const threatAssesmentFileResponse: any = [{
+    "@odata.type": "#microsoft.graph.fileAssessmentRequest",
+    "id": "18406a56-7209-4720-a250-08d772fccdaa",
+    "createdDateTime": "2019-11-27T05:44:00.4051536Z",
+    "contentType": "file",
+    "expectedAssessment": "block",
+    "category": "malware",
+    "status": "completed",
+    "requestSource": "administrator",
+    "fileName": "b3d5b715-4b88-4bbb-b0ae-9a9281a3f18a.csv",
+    "contentData": "",
+    "createdBy": {
+      "user": {
+        "id": "c52ce8db-3e4b-4181-93c4-7d6b6bffaf60",
+        "displayName": "Ronald Admin"
+      }
+    }
+  }];
+
+  const threatAssesmentUrlResponse: any = [{
+    "@odata.type": "#microsoft.graph.urlAssessmentRequest",
+    "id": "723c35be-8b5a-47ae-29c0-08d76ddb7f5b",
+    "createdDateTime": "2019-11-20T17:02:59.8160832Z",
+    "contentType": "url",
+    "expectedAssessment": "unblock",
+    "category": "phishing",
+    "status": "completed",
+    "requestSource": "administrator",
+    "url": "http://test.com",
+    "createdBy": {
+      "user": {
+        "id": "c52ce8db-3e4b-4181-93c4-7d6b6bffaf60",
+        "displayName": "Ronald Admin"
+      }
+    }
+  }];
+  //#endregion
+
+  let log: string[];
+  let logger: Logger;
+  let loggerLogSpy: sinon.SinonSpy;
+
+  before(() => {
+    commandInfo = Cli.getCommandInfo(command);
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    sinon.stub(pid, 'getProcessName').callsFake(() => '');
+    auth.service.connected = true;
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+    loggerLogSpy = sinon.spy(logger, 'log');
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get
+    ]);
+  });
+
+  after(() => {
+    sinonUtil.restore([
+      auth.restoreAuth,
+      telemetry.trackEvent,
+      pid.getProcessName
+    ]);
```

Comment:
Hi, just popping in here to say that we should refactor this `sinonUtil.restore([...` to `sinon.restore()`. This is a new way to ensure that the entire sandbox is cleared and will from now on be used in every command.